### ISSUE TITLE
Carousel default next/previous buttons text

### ISSUE THREAD

**Issue by aweell** [Telefonica/mistica-design#2276]
## Description

https://jira.tid.es/browse/DSNCORE-2252

In order not to repeat the title of the card all the time, the slide number should be mentioned in the pagination, for instance, "previous page, 1 of 4", "next page, 3 of 4", so that the user perceives the progression.

The user will still be able to focus within the carousel, which is the current behavior, and hear the title of each card with its corresponding number.

## Tasks

- [x] [Update specs](https://www.figma.com/design/UL4ALKbKMLbx89HXfbBHaZ/-2276---Carousel-default-next-previous-buttons-text?node-id=5841-4188&t=bmdL7DhzlZv4hJyn-4)
- [x] [Web ticket](https://jira.tid.es/browse/WEB-2288)
- [x] [UXW ticket](https://jira.tid.es/browse/PRODUCTDSN-3412)
- [x] [LOC ticket](https://jira.tid.es/browse/LOC-1917)
